doc/sal_constraints_and_recommendations.rst

```diff
@@ -372,6 +380,8 @@ The *<Subsystem>* and *<Alias>* tags for command's and logevent's must be consis
 
 ALL names must be less than 64 characters in length.
 
+.. _sal_constraints_and_recommendations:timestamps:
```

Comment:
At line 350 above "These should not have any embedded no alphanumeric characters" please fix no->non. Better yet reword that bit and the next one to just state that the only characters allowed are a-zA-Z0-9_ and that the name must start with a-z. I think it would be much clearer than trying to list all prohibited characters.

Reply:
Also the section "Ignored Attributes in Topics" is obsolete. If you are willing, please delete that section.

SAL 4.1 added the ability to define topics with no user-defined fields (and we are using 4.1.4, so we no longer need this old rule). Ignored fields are now deprecated.